docs/MigrationGuide.md

```diff
@@ -10,6 +11,7 @@ To see the biggest differences please consult the [changelog](Changelog.md).
 * `kafka.schemaRegistryCacheConfig` configuration entry was added - it was hardcoded before. 
   Default value of `kafka.schemaRegistryCacheConfig.availableSchemasExpirationTime` was changed from 1 minute to 10 seconds which will cause more often schema cache invalidation
 * [#3031](https://github.com/TouK/nussknacker/pull/3031) Attachments are now stored in database (see more in section `Other changes`). `attachmentsPath` was removed. Optional config `attachments.maxSizeInBytes` was introduced with default value of 10mb 
+* [#2992](https://github.com/TouK/nussknacker/pull/2992) deploySettings changed to deploymentCommentSettings, now when specified require you to also specify field validationPattern, specifying exampleComment is optional. 
```

Comment:
And what about changes in `commentSettings`?

Reply:
forgot about them, thanks